ref/cats_2019-09-16-database-design.sql

```diff
@@ -28,6 +28,7 @@ DROP TABLE IF EXISTS `breed`;
 CREATE TABLE `breed` (
   `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
   `breed` varchar(255) NOT NULL,
+  `favourite_id` int(11) unsigned DEFAULT NULL,
```

Comment:
needs to be a foreign key
Since this is just a script, you might want to use this in your scrape instead of having it duplicated, where you have to maintain it in two places - or swap this for more informal structural documentation so people don't use it locally?